README.rst

```diff
@@ -109,7 +109,7 @@ Running Simulations
 
 You can then run simulations by, e.g.::
 
-   (vivarium_census_prl_synth_pop) :~$ simulate run -v src/vivarium_census_prl_synth_pop/model_specifications/model_spec.yaml
+   (vivarium_census_prl_synth_pop) :~$ simulate run --pdb -v src/vivarium_census_prl_synth_pop/model_specifications/model_spec.yaml
```

Comment:
This is atypical for our readmes. Is there a reason you're adding this here? Even if we want to make this change, PRs should be targeted and not include unrelated changes

Reply:
oops, unintentional inclusion.

```suggestion
   (vivarium_census_prl_synth_pop) :~$ simulate run -v src/vivarium_census_prl_synth_pop/model_specifications/model_spec.yaml
```